code/controllers/subsystem/ipintel.dm

```diff
@@ -0,0 +1,14 @@
+SUBSYSTEM_DEF(ipintel)
```

Comment:
Add the `INIT_ORDER_XKEYSCORE` here

Reply:
Does it need its own init order?
If it does, why?
Just asking.

Reply:
I mean if were going on `I ported it like this because that's how TG does it.`, then surely it should have it. I dont /think/ it makes a huge difference, but we have the defines for a reason.

Reply:
Ported